File: atividade_nicolas/alunos/nicolas_g1/icaro/avaliacao1 - javascript/dollar.js
import { input, print } from "../io_utils.js"

function main() {

    const cotacao = Number(input("digite a cotação atual do dolar em reais (cotacao em 06/05 era R$ 5,08): "));
    const valor_a_comprar = Number(input("digite quantos dólares você deseja comprar: "));
    print(calcula_valor_total(cotacao, valor_a_comprar));
    
}

function calcula_valor_total(cotacao, valor_a_comprar) {

    const IOF = (1.1/100) * valor_a_comprar;
    const total = (valor_a_comprar + IOF) * cotacao;

    return `o valor total a ser pago é R$ ${total}`;

}

main();

File: atividade_nicolas/alunos/nicolas_g1/kleberson/dollar.py
cotacao_atual_dolar = 5.08
qtd_dolar_para_comprar = int(input('Informe a quantidade de dólar a ser comprado: '))
iof = qtd_dolar_para_comprar * 0.0011
valor_a_pagar_em_reais = iof * 5.08 

print('O valor a ser pago em reais é: {} R$'.format(valor_a_pagar_em_reais))
print(iof)

#valor do dolar a 5.08 -- comprar 200 dólares
#vou pagar 1000 e alguma coisa